easydita-knowledge-portal/functions.php

```diff
@@ -128,7 +128,7 @@ function easydita_knowledge_portal_scripts() {
 function easydita_knowledge_portal_change_style_version_num($styles) {
 	$styles -> default_version = "123";
 }
-//add_action("wp_default_styles", "easydita_knowledge_portal_change_style_version_num");
+add_action("wp_default_styles", "easydita_knowledge_portal_change_style_version_num");
```

Comment:
Since this was the only change in this file, you should just check it out from the base; the other changes are whitespace.